SampleApp/Sources/RootViewController.swift

```diff
@@ -24,6 +24,9 @@ final class RootViewController : UIViewController
             DemoItem(title: "Keyboard Scrolling", onTap: { [weak self] in
                 self?.push(ScrollViewKeyboardViewController())
             }),
+            DemoItem(title: "Responsive Layout", onTap: { [weak self] in
```

Comment:
My only nit would be to mention "GeometryReader" in this title so someone looking for an example would notice it!